de.fin_ger.OvGUCanteen.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "de.fin_ger.OvGUCanteen",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "gnome-ovgu-canteen",
```

Comment:
Despite it's being  just the executable name, I don't think it make sense to name it gnome-*. It's completely unrelated to GNOME.

Reply:
Yes, it's not related to gnome. It's just made for the gnome desktop and works best with it.

Reply:
In such case, the categories in the desktop file makes more sense. There's no reason to keep with the trend of naming everything that's built with GTK called gnome-* as it's not part of the GNOME project.
This is not a blocker of course, but just a nitpick

Reply:
I can of course rename the project. Would something like gtk-ovgu-canteen be ok? I plan to build apps for other platforms and would like to avoid namimg collisions.

Reply:
`ovgu-canteen-gtk` sounds like the way to go if you're planning to have different versions using different toolkits

Reply:
I renamed the project